csharp/linq.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+					
+public class Program
+{
+	public static void Main()
+	{
+		Enumerable
+			.Range(1,100)
+			.Select(FizzBuzz.FromNumber)
```

Comment:
This is sort of silly on my part, but I didn't realize you could use public static members as Funcs. I generally consider them to be functions, but I didn't realize that C# agreed with me.